### PR TITLE
Fix some issues, new attack icon (from dota2), login icon

### PR DESCRIPTION
**Pull request by blukai**

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -21,29 +21,35 @@ import BurgerMenu from '../BurgerMenu';
...
+};
 
 const LogoGroup = ({ width }) => {
-  if (width <= mobile) {
-    navbarPages.push({
-      name: strings.app_report_bug,
-      external: true,
-      path: 'https://github.com/odota/ui/issues',
-    });
+  if (width < mobile && navbarPages.map(page => page.path === '//github.com/odota/ui/issues').filter(Boolean).length < 1) {
```
Comment (howardchung) [inline src/components/Header/Header.jsx]: What if we make "report bug" part of the array permanently?  That would move it to the left side, but it's cleaner code than this splicing.

Comment (blukai) [inline src/components/Header/Header.jsx]: Noooo 😛

Comment (howardchung) [inline src/components/Header/Header.jsx]: How about:

* Save the length of the array
* Push if the navBarPages.length === savedLength

Comment (blukai) [inline src/components/Header/Header.jsx]: Good, but how?

File: src/components/Header/Header.jsx [hunk 2]
```diff
@@ -21,29 +21,38 @@ import BurgerMenu from '../BurgerMenu';
...
-    navbarPages.push({
-      name: strings.app_report_bug,
-      external: true,
-      path: 'https://github.com/odota/ui/issues',
-    });
+  if (width < mobile && navbarPagesLength === navbarPages.length) {
+    navbarPages.push(navbarBug);
+  }
+
+  // If `bug` item in navbar and it's becoming wider, remove `bug` from navbarPages
```
Comment (howardchung) [inline src/components/Header/Header.jsx]: Actually how about:
* Define a new const that's a copy of navbarPages in this function's scope
* Push report bug onto it if width is < mobile
* That way we avoid mutating navbarPages and having to do the check, as well as avoiding the need to splice

Comment (howardchung) [inline src/components/Header/Header.jsx]:
```
const extendedNavBar = [...navBarPages];
if (width < mobile) {
extendedNavBar.push(navbarBug);
}
```

**Comment by blukai** [comment]
@howardchung, a bit different way, without push 🦄

**Pull request merged**